doc/manual/cookbooks/choosing_nonlinear_solver_scheme/doc.tex

```diff
@@ -0,0 +1,326 @@
+When the problem to solve in \aspect{} is nonlinear, choosing the right nonlinear 
+solver scheme can be crucial to get fast convergence to the correct solution or  
+even to converge at all. This section is aimed to explain the differences between 
+the three approaches available in \aspect{}: iterated Stokes, iterated defect correction
+Stokes and Newton Stokes. The actual nonlinear solver you choose also will state 
+which advection solver you choose, but we won't discuss that in this section.
+
+This cookbook will be structured by first explaining the terms that will be used by looking 
+at a linear system and then explaining the three solver options very 
+generically through looking at solving for $\sin(x)=0$. Then the options for the Newton
+solver will be explained, including how the stabilization works, and we end with 
+the results to some simple test cases.
+
+\paragraph{Soveling a linear system}
+
+When we try to solve a system of equations we can write it down as:
+\begin{align}
+    \label{sec:cnss:eq1:axb}
+    A x = b
+\end{align}, where $A$ is a know value or matrix, $x$ is a unknown scalar or vector of
+ knowns, and $b$ is the right hand side scalar/vector. Now if we want to know what $x$ is 
+ for a given $A$ and $b$, we can rearrange the equation to:
+\begin{align}
+    x = A^{-1} b
+\end{align}, where $A^{-1}$ is the inverse of $A$. In 1D a concrete example would be $A=2$,
+$b=5$, we would get:
+\begin{align}
+    2 x = 6 \\
+    x = 2^{-1} 6 \\
+    x = \frac{1}{2} 6 \\
+    x = 3
+\end{align}
+
+\paragraph{Sovling $\bf \sin(x)=0$}
+
+Making the problem non-linear in our context means that $A$ becomes dependent on $x$. We 
+can write this down as $A(x)$. So our equation becomes:
+\begin{align}
+    \label{sec:cnss:eq1:axxb}
+    A(x) x = b
+\end{align}. To solve this nonlinear equation we need to iterate. To illustrate this we are
```

Comment:
Put the dot at the end of the formula. Here, it shows up at the front of the next line.